ModelFitting/ModelFitting/Models/CompactSersicModel.h

```diff
@@ -16,12 +16,22 @@ template <typename ImageType>
 class CompactSersicModel : public CompactModelBase<ImageType> {
 
 public:
+
+  enum class SamplingMethod {
```

Comment:
No use for this, or you have left it as a placeholder?

Reply:
Will fix